Dockerfile

```diff
@@ -89,10 +89,10 @@ COPY test/cni_plugin_helper.bash /opt/cni/bin/cni_plugin_helper.bash
 ENV CRICTL_COMMIT release-1.11
 RUN set -x \
        && export GOPATH="$(mktemp -d)" \
-       && git clone https://github.com/kubernetes-incubator/cri-tools.git "$GOPATH/src/github.com/kubernetes-incubator/cri-tools" \
```

Comment:
travis' failing here, we need to keep cri-tools in incubator path as master does :) easy fix